common/src/main/java/com/hedera/block/common/utils/MathUtilities.java

```diff
@@ -19,5 +19,15 @@ public static boolean isPowerOfTwo(final int toCheck) {
         return (0 < toCheck) && ((toCheck & (toCheck - 1)) == 0);
     }
 
+    /**
+     * This method checks if the given number is even.
+     *
+     * @param toCheck the number to check if it is even
+     * @return {@code true} if the given number is even
+     */
+    public static boolean isEven(final int toCheck) {
+        return (toCheck % 2) == 0;
```

Comment:
You might consider the following slightly more reliable option: 
```suggestion
        return (toCheck & 1) == 0;
```

Reply:
Thank you for the suggestion!!

✅ Done!